### PR TITLE
[NETBEANS-3097]:Corrected font-color type name for text-block(JDK-13)

### PR DESCRIPTION
**Pull request by arusinha**

### CODE REVIEW

**Comment by neilcsmith-net** [comment]
Out of interest, why is this capitalized with a space when the others are lower case and hyphenated?

**Comment by geertjanw** [comment]
Agreed. Needs to be registered the same way as for all the others.

**Comment by arusinha** [comment]
Hi @neilcsmith-net / @geertjanw  should I retain the old name text-block ?

**Comment by neilcsmith-net** [comment]
@arusinha that depends what your reason for changing it is?  What does "corrected" mean?  To make it work or to make it look better?

**Comment by arusinha** [comment]
Name was changed to make it look better

**Comment by geertjanw** [comment]
The display name should be Text Block, the programmatic name text-block. See how it is done for the others.

**Comment by jlahoda** [comment]
Should be possible to put text-block=Text Block to the Bundle.properties in the same directory.

**Comment by neilcsmith-net** [comment]
Looks good to me.  No need for the cherry picking label - we no longer cherry pick!  When merged it will be in 11.2.

File: java/java.editor/src/org/netbeans/modules/java/editor/resources/fontsColors.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
     <fontcolor name="mod-protected" />
     <fontcolor name="mod-public" />
     <fontcolor name="mod-keyword" default="keyword" />
-    <fontcolor name="text-block" bgColor="EEDDDD" />
+    <fontcolor name="mod-unindented-text-block" bgColor="EEDDDD" />
```
Comment (arusinha) [inline java/java.editor/src/org/netbeans/modules/java/editor/resources/fontsColors.xml]: had reverted back to old name "mod-unindented-text-block"

**Pull request merged**